src/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateTimeConverter.cs

```diff
@@ -118,6 +121,33 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 return dt.ToString(format, CultureInfo.CurrentCulture);
             }
 
+            if (destinationType == typeof(InstanceDescriptor) && value is DateTime) 
+            {
+                DateTime dt = (DateTime)value;
+                
+                if (dt.Ticks == 0) 
+                {
+                    // Make a special case for the empty DateTime
```

Comment:
`// Make a special case for the empty DateTime` -> `// Special case for empty DateTime`